deployment/tfhub/object-detection/base64/object-detection-base64-deploy.ipynb

```diff
@@ -0,0 +1,390 @@
+   "outputs": [],
+   "source": [
+    "client = Client(VERTA_HOST)\n",
+    "project = client.set_project(PROJECT_NAME)\n",
+    "registered_model = client.get_or_create_registered_model(\n",
+    "    name = PROJECT_NAME, \n",
+    "    labels = ['object-detection']\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input = {\n",
+    "    \"file_name\": \"\",\n",
+    "    \"image_str\": \"\"\n",
+    "}\n",
+    "output = {\n",
+    "    \"file_name\": \"\",\n",
+    "    \"has_car\": False,\n",
+    "    \"score\": 0,\n",
+    "    \"ymin\": 0,\n",
+    "    \"xmin\": 0,\n",
+    "    \"ymax\": 0,\n",
+    "    \"xmax\": 0\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = registered_model.create_standard_model(\n",
+    "    model_cls = DetectObject,\n",
+    "    environment = Python(requirements = ['tensorflow', 'tensorflow_hub', 'dill', 'Pillow', 'wget']),\n",
+    "    model_api = ModelAPI([input], [output]),\n",
+    "    name = MODEL_NAME\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## VM Auto Scaling and Resources Configuration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "autoscaling = Autoscaling(min_replicas = 1, max_replicas = 20, min_scale = 0.1, max_scale = 10)\n",
+    "autoscaling.add_metric(CpuUtilizationTarget(0.6))\n",
+    "autoscaling.add_metric(MemoryUtilizationTarget(0.7))\n",
+    "autoscaling.add_metric(RequestsPerWorkerTarget(1))\n",
+    "resources = Resources(cpu = 2., memory = '12Gi')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Endpoint Creation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint = client.get_or_create_endpoint(ENDPOINT_NAME)\n",
+    "status = endpoint.update(\n",
+    "    model, \n",
+    "    strategy = DirectUpdateStrategy(),\n",
+    "    autoscaling = autoscaling,\n",
+    "    resources = resources,\n",
+    "    wait = True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prediction Test"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "assert status['status'] == \"active\"\n",
+    "url = 'http://s3.amazonaws.com/verta-starter/street-view-images/000001_0.jpg'\n",
+    "file = url.split('/')[-1]\n",
+    "wget.download(url, file)\n",
+    "\n",
+    "with open(file, 'rb') as img:\n",
+    "    img_bytes = base64.b64encode(img.read())\n",
+    "    img_str = img_bytes.decode('utf-8')\n",
+    "    img_str = json.dumps(img_str)\n",
+    "    img_str = np.array(img_str).tolist()\n",
```

Comment:
We do not need this line because `json.dumps(img_str)` is already compatible with `get_deployed_model().predict()`! Removing it still works, based on my testing.